packages/composite-checkout/src/lib/payment-methods/registered-methods.js

```diff
@@ -22,6 +22,7 @@ export default function loadPaymentMethods() {
 	registerPaymentMethod( {
 		id: 'apple-pay',
 		LabelComponent: ApplePayLabel,
+		summaryText: 'Apple Pay',
```

Comment:
It's better (and might be required by some libraries) to use the `localize()` call here. Obviously that doesn't work because it's not available in this context, but we could make this argument take a React element type instead and then we'd have access to the hook and be able to do the translation in that component.